sources/packages/backend/apps/api/src/services/student-assessment/student-assessment.service.ts

```diff
@@ -155,9 +157,11 @@ export class StudentAssessmentService extends RecordDataModelService<StudentAsse
         "assessment.id",
         "application.id",
         "application.applicationStatus",
+        "currentAssessment.id",
       ])
       .innerJoin("assessment.application", "application")
       .innerJoin("application.student", "student")
+      .leftJoin("application.currentAssessment", "currentAssessment")
```

Comment:
`leftJoin` can be changed to an `innerJoin`.

Reply:
Updated